code/modules/vehicle/bicycle.dm

```diff
@@ -0,0 +1,62 @@
+/obj/vehicle/bike
+	name = "bicycle"
+	desc = "Two wheels of FURY!"
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "bicycle"
+	vehicle_move_delay = 1
```

Comment:
Ok but what if badass people were like slightly faster on a bike

Reply:
Hmm, maybe in another PR since you could apply that logic to other vehicles too.

Reply:
Fair enough! Lewc Vehicle rework 2023!